README.md

```diff
@@ -12,6 +12,9 @@ My goal here is to create the first page of a mobile-friendly portfolio page.
 
 ## Live Demo
 ![screenshot](Images/Template_2_sample.PNG)
+(https://raw.githack.com/adamteddychang/Hello-Portfolio/Desktop-portfolio-2/index.html)
```

Comment:
- [ ] Make sure your live link is pointing to the current work. Now it is broken.
![Screenshot from 2021-09-23 20-22-54](https://user-images.githubusercontent.com/65860678/134556547-99222352-b17f-43f7-8bbd-cb112d752992.png)

Reply:
Sorry about that, looking at the live demo link from my end is a little distorted but not as broken as the picture showed above. The screenshot from my end is shown below. Thank you for the catch. 
![Live-demo](https://user-images.githubusercontent.com/63560332/134560220-5d71db68-410d-4457-ada5-e20e1be90e2f.PNG)
.

Reply:
This is how it looks like if the webpage is opened locally.
![local-demo](https://user-images.githubusercontent.com/63560332/134560451-0c161b66-8b63-41b1-9327-337b72e6b744.PNG)